docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,52 @@
+Read the Docs for Business subscription management
+==================================================
```

Comment:
Good to get this in now, it is an important page to be able to handle in the new navigation structure.